### ISSUE TITLE
Rules SA1117, SA1115 and SA1113 are triggered for out parameter with inline typing

### ISSUE THREAD

**Issue by JamesHough** [StyleCop/StyleCop#161]
The rules SA1117 and SA1115 are being triggered for the following code (VS 2017, StyleCop 5.0.6419.0)

```
return report.Render(
    this.Options.FileFormat,
    this.Options.FileMetadata,
    out string mimeType,
    out string encoding,
    out string fileNameExtension,
    out string[] streams,
    out Warning[] warnings);
```

The StyleCop output is

> SA1117 : CSharp.Readability : All method parameters must be placed on the same line, or each parameter must be placed on a separate line.
> SA1115 : CSharp.Readability : The parameter must begin on the line after the previous parameter.
> SA1115 : CSharp.Readability : The parameter must begin on the line after the previous parameter.
> SA1115 : CSharp.Readability : The parameter must begin on the line after the previous parameter.
> SA1115 : CSharp.Readability : The parameter must begin on the line after the previous parameter. 
> SA1113 : CSharp.Readability : The comma must be placed on the same line as the previous parameter.	

The rules above will only accept code that is written in the following format:

```
string mimeType, encoding, fileNameExtension;
string[] streams;
Warning[] warnings;

return report.Render(
    this.Options.FileFormat,
    this.Options.FileMetadata,
    out mimeType,
    out encoding,
    out fileNameExtension,
    out streams,
    out warnings);
```

**Comment by NageshAndani** [comment]
I am getting Rule **SA1126** triggered for out parameter with inline declaration.

**Comment by ravithakkar85** [comment]
> I am getting Rule **SA1126** triggered for out parameter with inline declaration.
> 
> `myObject.GetData(out bool isLoggedIn);`
> 
> Stylecop's Rule viloation error is:
> 
> > SA1126 : CSharp.Readability : The call to bool must begin with the 'this.', 'base.', 'object.' or 'LoginState.' or 'BaseLogin.' prefix to indicate the intended method call.

I am getting similar error. What's the fix for this stylecop error @NageshAndani ?
